doc/user/content/sql/drop-index.md

```diff
@@ -6,50 +6,40 @@ menu:
     parent: 'commands'
 ---
 
-`DROP INDEX` removes an index from a materialized view. (Non-materialized views do not have any indexes.)
+`DROP INDEX` removes the specified index.
 
 ## Syntax
 
-{{< diagram "drop-index.svg" >}}
```

Comment:
Do you mind removing the `.svg` and maybe also the syntax entry from the grammar file when you replace a railroad diagram?